src/org/ods/component/ScanWithSonarStage.groovy

```diff
@@ -253,12 +254,19 @@ class ScanWithSonarStage extends Stage {
         }
     }
 
-    private createBitbucketCodeInsightReport(String qualityGateResult, String nexusUrlReport, String sonarProjectKey) {
-        String sorQubeScanUrl = sonarQube.getSonarQubeHostUrl() + "/dashboard?id=${sonarProjectKey}"
+    private createBitbucketCodeInsightReport(
+        String qualityGateResult, String nexusUrlReport,String sonarProjectKey, String edition, String branch) {
+        String sorQubeScanUrl = ""
         String title = "SonarQube"
         String details = "Please visit the following links to review the SonarQube report:"
         String result = qualityGateResult == "OK" ? "PASS" : "FAIL"
 
+        if (edition != 'community') {
+            sorQubeScanUrl = sonarQube.getSonarQubeHostUrl() + "/dashboard?id=${sonarProjectKey}&branch=${branch}"
+        } else {
+            sorQubeScanUrl = sonarQube.getSonarQubeHostUrl() + "/dashboard?id=${sonarProjectKey}"
```

Comment:
You can initialize the variable with this value and, if it's not community, then concat the branch.